src/content/docs/en/pages/devtools/cli/azion-cli/commands/create/create.mdx

```diff
@@ -563,3 +562,37 @@ The `--file` option specifies the path to a JSON file containing the attributes
 
 The `--help` option displays more information about the 'create edge-storage bucket' command.
 
+---
+
+## Edge Storage object 
+
+#### Usage
+
+```bash
+$ azion create edge-storage object --bucket-name 'mynewbucket' --object-key 'path/to/my/remote/file.txt' --source './local/file.txt'
+```
+
+#### Required flags
+
+##### bucket-name
+
+The `--bucket-name` flag sets the name of the Edge Storage bucket where the object will be stored.
+
+##### object-key
+
+The `--object-key` flag sets the name of the object to be stored in the bucket. It can also be used to indicate a full file path (like 'path/to/file/bucket/file.txt').
```

Comment:
```suggestion
The `--object-key` flag sets the name of the object to be stored in the bucket. It can also be used to indicate a full file path (such as 'path/to/file/bucket/file.txt').
```